tools/api/generate_go_protobuf.py

```diff
@@ -47,9 +47,14 @@ def generate_protobufs(output):
         #
         # Example output directory:
         # go_out/envoy/config/bootstrap/v2
-        rule_dir, proto = rule.decode()[len('@envoy_api//'):].rsplit(':', 1)
-        input_dir = os.path.join(
-            bazel_bin, 'external', 'envoy_api', rule_dir, proto + '_', IMPORT_BASE, rule_dir)
+        _, stripped_path = rule.decode().rsplit('//', 1)  # envoy/config/bootstrap/v3:pkg_go_proto
+        rule_dir, proto = stripped_path.rsplit(':', 1)
```

Comment:
nit: as `stripped_path` is only used in the next line - why not just do this with

```
rule_dir, proto = rule.decode().rsplit('//', 1)[1].rsplit(":", 1)
```